src/vs/workbench/api/common/configurationExtensionPoint.ts

```diff
@@ -119,9 +119,24 @@ const configurationEntrySchema: IJSONSchema = {
 							tags: {
 								type: 'array',
 								items: {
-									type: 'string'
+									type: 'string',
+									enum: [
+										'accessibility',
+										'advanced',
+										'experimental',
+										'telemetry',
+										'usesOnlineServices',
+									],
+									enumDescriptions: [
+										nls.localize('accessibility', 'Accessibility settings'),
+										nls.localize('advanced', 'Advanced settings are hidden by default in the Settings editor unless the user chooses to show advanced settings.'),
+										nls.localize('experimental', 'Experimental settings'),
+										nls.localize('telemetry', 'Telemetry settings'),
+										nls.localize('usesOnlineServices', 'Settings that use online services')
```

Comment:
We have some more tags accessible through the filter icon on the right of the Settings editor input. Would it make sense to add `preview` considering there's also `experimental`?

<img width="701" height="341" alt="Settings editor tags" src="https://github.com/user-attachments/assets/96866c5c-e1e0-4bcb-bdef-644f4087af8f" />